pinot-spi/src/main/java/org/apache/pinot/spi/config/table/FieldConfig.java

```diff
@@ -85,7 +85,7 @@ public FieldConfig(@JsonProperty(value = "name", required = true) String name,
 
   // If null, there won't be any index
   public enum IndexType {
-    INVERTED, SORTED, TEXT, FST, H3, JSON, RANGE
+    INVERTED, SORTED, TEXT, FST, NATIVE_FST, H3, JSON, RANGE
```

Comment:
Suggest treating native FST as FST, and add an enum in `IndexingConfig` for the FST version (`LUCENE` or `NATIVE`)

Reply:
I do not think that would work, since we do not allow specifying sub config per field in IndexingConfig?